m3inference/m3twitter.py

```diff
@@ -193,6 +193,34 @@ def _twitter_api(self,id=None,screen_name=None):
         return self.process_twitter(r.json())
 
 
+    def _twitter_api_lookup(self, ids=None, batch_size=16, num_workers=4):
```

Comment:
We probably could reuse/extend the `twitter_api` function by extending `id` to `ids`?